zng/resolver/context.go

```diff
@@ -429,6 +429,15 @@ func (c *Context) parseRecordTypeBody(in string) (string, zng.Type, error) {
 		return "", nil, zng.ErrTypeSyntax
 	}
 	var columns []zng.Column
+	in, ok = match(in, "]")
+	if ok {
+		typ, err := c.LookupTypeRecord([]zng.Column{})
+		if err != nil {
+			return "", nil, err
+		}
+		return in, typ, nil
+	}
+
```

Comment:
Please don't insert code that doesn't use `columns` between its declaration and the code that does use it.